src/fhir/2_CarePlan.patient-dcw-jt.json

```diff
@@ -0,0 +1,99 @@
+{
+    "resourceType": "CarePlan",
+    "id": "CarePlan-patient-dcw-jt",
+    "description": "Hand-coded CarePlan for patient-dcw-jt",
+    "status": "active",
+    "intent": "order",
+    "category": [
+        {
+            "coding": [
+                {
+                    "system": "http://snomed.info/sct",
+                    "code": "719091000000102",
+                    "display": "Questionnaire"
+                }
+            ],
+            "text": "Questionnaire"
+        }
+    ],
+    "subject": {
+        "reference": "Patient/patient-dcw-jt"
+    },
+    "activity": [
+        {
+            "detail": {
+                "instantiatesCanonical": [
+                    "Questionnaire/CIRG-ECOG-SELF-REPORT"
```

Comment:
@mcjustin I think this is CIRG-ECOG12 (see [here](https://github.com/uwcirg/asbi-screening-app/blob/460bcabca655486f6e8a6e5678f3acef6a1ca4d9/src/fhir/1_Questionnaire_ECOG12.json#L3))  I just realize that [this one](https://github.com/uwcirg/asbi-screening-app/blob/master/src/fhir/1_Questionnaire-ECOG.json) is incorrect, as it is the original long form,  and should have been removed.  I will do a separate PR to remove that one.

Reply:
Thanks for catching that - fixed now.